### PR TITLE
Fix tox

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,6 +2,29 @@
 Resolwe Bioinformatics Python API
 =================================
 
+|build| |docs| |pypi_version| |pypi_pyversions| |pypi_downloads|
+
+.. |build| image:: https://travis-ci.org/genialis/resolwe-bio-py.svg?branch=master
+    :target: https://travis-ci.org/genialis/resolwe-bio-py
+    :alt: Build Status
+
+.. |docs| image:: https://readthedocs.org/projects/resolwe-bio-py/badge/?version=latest
```
Comment (tjanez) [inline README.rst]: @dblenkus, please change `readthedocs.org` to `readthedocs.io` to follow [Read the Docs subdomains hosting change](http://blog.readthedocs.com/securing-subdomains/). We'll also have to update other projects eventually.

File: README.rst [hunk 2]
```diff
@@ -2,6 +2,29 @@
...
+    :target: http://resolwe-bio-py.readthedocs.org/
```
Comment (tjanez) [inline README.rst]: Same as above.

**Comment by tjanez** [comment]
Otherwise, good job!

**Comment by dblenkus** [comment]
Fixed.

**Comment by mstajdohar** [comment]
LGTM

**Pull request merged**